themes/default/content/docs/pulumi-cloud/deployments/reference.md

```diff
@@ -1,7 +1,7 @@
 ---
 title_tag: "Using Pulumi Deployments"
 meta_desc: Reference documentation for configuring and using Pulumi Deployments
-title: "Using deployments"
+title: "Using Pulumi Deployments"
```

Comment:
this will not fit in the navigation, you are welcome to adjust the title_tag, but changes to titles will make the nav more difficult to use.